src/branding-bar/BrandingBar.js

```diff
@@ -1,8 +1,62 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-const BrandingBarContainer = styled.header``;
+import LogoIconSrc from "../assets/icons/recidiviz_logo.svg";
+import { LinkPill } from "../pill";
+
+const brandingBarFlexProperties = css`
+  align-items: center;
+  display: flex;
+`;
+
+const BrandingBarContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 50px;
+`;
+
+const BrandingBarHeader = styled.div`
+  ${brandingBarFlexProperties}
+`;
+
+const Logo = styled.img``;
+
+const BrandingBarTitle = styled.h1`
+  color: ${(props) => props.theme.colors.heading};
+  font: ${(props) => props.theme.fonts.display};
+  display: inline;
+  margin-left: 16px;
+`;
+
+const BrandingBarLinkContainer = styled.div``;
```

Comment:
Are we roughly trying to avoid using HTML elements in our render functions and instead trying to name all of our markup elements even if their styles are otherwise empty like this one?

Reply:
That does appear to be our convention, yes. Let's keep following it.